.github/workflows/integration_test.yaml

```diff
@@ -12,4 +12,8 @@ jobs:
       juju-channel: 3/stable
       self-hosted-runner: true
       charmcraft-channel: latest/edge
-      modules: '["test_action.py", "test_config.py", "test_charm.py", "test_http_interface.py", "test_cos.py", "test_ingress.py", "test_ha.py"]'
+      modules: '["test_action.py", "test_config.py", "test_charm.py", "test_http_interface.py", "test_cos.py", "test_ingress.py", "test_ha.py", "test_haproxy_route.py"]'
```

Comment:
Can you order this alphabetically? IT'll be easy to see if any test is missing or fine one if we want to remove it, etc

Reply:
thanks, updated to make it more readable and ordered them alphabetically